### PR TITLE
Spelling src bootstrap

### PR DESCRIPTION
**Pull request by jsoref**
The various src/* items seem slightly disparate, so I'm doing src/* individually.

split from #110392

### CODE REVIEW

File: src/bootstrap/llvm.rs [hunk 1]
```diff
@@ -1153,7 +1153,7 @@ impl Step for Libunwind {
         run.builder.ensure(Libunwind { target: run.target });
     }
 
-    /// Build linunwind.a
+    /// Build libunwind.a
```
Comment (jsoref) [inline src/bootstrap/llvm.rs:1156]: this is the main notable item in the set

File: src/bootstrap/sanity.rs [hunk 2]
```diff
@@ -100,7 +100,7 @@ pub fn check(build: &mut Build) {
 Couldn't find required command: cmake
 
 You should install cmake, or set `download-ci-llvm = true` in the
-`[llvm]` section section of `config.toml` to download LLVM rather
+`[llvm]` section of `config.toml` to download LLVM rather
```
Comment (jsoref) [inline src/bootstrap/sanity.rs:103]: My double corrections have been hit or miss. I hope I'm right but I wouldn't bet on it.

**Comment by jsoref** [comment]
@albertlarsan68: are you waiting for something from me?

**Comment by albertlarsan68** [comment]
Thanks for the PR!
@bors r+ rollup

**Pull request merged**